Linting broken in Webstorm

Hello,

After updating to eslint-plugin-eslint-comments@3.0.0 I have a problem in Webstorm 2018.1.5

```
Error: Cannot find module '/Applications/WebStorm.app/Contents/plugins/JavaScriptLanguage/languageService/eslint/'
Referenced from: /Users/akopersky/WebstormProjects/WebCC/tools/eslint-config-base/index.js
Referenced from: /Users/akopersky/WebstormProjects/WebCC/tools/eslint-config-web/index.js
Referenced from: /Users/akopersky/WebstormProjects/WebCC/tools/eslint-config-react/index.js
Referenced from: /Users/akopersky/WebstormProjects/WebCC/applications/alta-go-cc/src/main/.eslintrc.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:571:15)
    at Function.Module._load (internal/modules/cjs/loader.js:497:25)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Array.from.map.eslintPath (/Users/akopersky/WebstormProjects/WebCC/node_modules/eslint-plugin-eslint-comments/lib/internal/get-linters.js:18:54)
```

I dont know if this problem on Webstorm side. But temporary I fix it wit this code in https://github.com/mysticatea/eslint-plugin-eslint-comments/blob/master/lib/internal/get-linters.js#L13

`.filter(id => !id.includes('WebStorm.app'))`

--- a/lib/internal/get-linters.js
+++ b/lib/internal/get-linters.js
@@ -5,7 +5,7 @@
 "use strict"
 
 const path = require("path")
-const needle = `${path.sep}eslint${path.sep}`
+const needle = `${path.sep}node_modules${path.sep}eslint${path.sep}`
 
 module.exports = () => {
     const eslintPaths = new Set(
@@ -13,5 +13,21 @@ module.exports = () => {
             .filter(id => id.includes(needle))
             .map(id => id.slice(0, id.indexOf(needle) + needle.length))
     )
-    return Array.from(eslintPaths).map(eslintPath => require(eslintPath).Linter)
+    const linters = []
+
+    for (const eslintPath of eslintPaths) {
+        try {
+            const linter = require(eslintPath).Linter
+
+            if (linter) {
+                linters.push(linter)
+            }
+        } catch (error) {
+            if (error.code !== "MODULE_NOT_FOUND") {
+                throw error
+            }
+        }
+    }
+
+    return linters
 }

